### ISSUE TITLE
Not working on B7D-Z 06.10.30 :(

### ISSUE THREAD

**Issue by MrAshen** [RootMyTV/RootMyTV.github.io#89]
Hey Guys, 

my TV has updated from 06.10.01 - on which it worked perfectly - to 06.10.30. The exploit no longer works. Is there any way to get it working again?

thanks for your great Work! 🙂

**Comment by pandalec** [comment]
same here...

**Comment by meanwhile28** [comment]
Same here.. any idea if this can be fixed or is there any other way to root the tv ?

**Comment by BitHappy** [comment]
Same! I'm so sad! I had no idea this was even possible until today and I feel like I just updated to 6.10.30 recently.. I'm such an idiot.. Hoping for a fix soon? :( And until then I'm not updating my TV ever again!

**Comment by crapzDK** [comment]
yeah fucked myself as well :( OLED55B7V-Z  06.10.30

**Comment by throwaway96** [comment]
RootMyTV is patched on that firmware version.

Since you're not running webOS 4 or later, [crashd](https://gist.github.com/throwaway96/e811b0f7cc2a705a5a476a8dfa45e09f) won't work. There is a [hardware method](https://gist.github.com/throwaway96/827ff726981cc2cbc46a22a2ad7337a1) for enabling debug mode, which would allow you to get root.

**Comment by crapzDK** [comment]
yeah, not even able to downgrade the firmware from OLED55B7V-Z 06.10.30 right ? 
or i had no luck with it :D
